scripts/pi-hole/php/header.php

```diff
@@ -185,152 +74,43 @@ function getTemperature()
         html { background-color: #000; }
     </style>
 <?php } ?>
-    <link rel="stylesheet" href="style/vendor/SourceSansPro/SourceSansPro.css?v=<?php echo $cacheVer; ?>">
+
+    <!-- Common styles -->
     <link rel="stylesheet" href="style/vendor/bootstrap/css/bootstrap.min.css?v=<?php echo $cacheVer; ?>">
+<?php if ($auth) { ?>
     <link rel="stylesheet" href="style/vendor/datatables.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/datatables_extensions.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/daterangepicker.min.css?v=<?php echo $cacheVer; ?>">
+<?php } ?>
     <link rel="stylesheet" href="style/vendor/AdminLTE.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/select2.min.css?v=<?php echo $cacheVer; ?>">
 
 <?php if (in_array($scriptname, array('groups.php', 'groups-adlists.php', 'groups-clients.php', 'groups-domains.php'))) { ?>
+    <!-- Group management styles -->
     <link rel="stylesheet" href="style/vendor/animate.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/bootstrap-select.min.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/vendor/bootstrap-toggle.min.css?v=<?php echo $cacheVer; ?>">
 <?php } ?>
+
+    <!-- Theme styles -->
     <link rel="stylesheet" href="style/pi-hole.css?v=<?php echo $cacheVer; ?>">
     <link rel="stylesheet" href="style/themes/<?php echo $theme; ?>.css?v=<?php echo $cacheVer; ?>">
+
     <noscript><link rel="stylesheet" href="style/vendor/js-warn.css?v=<?php echo $cacheVer; ?>"></noscript>
 
+    <!-- scripts -->
     <script src="scripts/vendor/jquery.min.js?v=<?php echo $cacheVer; ?>"></script>
     <script src="style/vendor/bootstrap/js/bootstrap.min.js?v=<?php echo $cacheVer; ?>"></script>
     <script src="scripts/vendor/adminlte.min.js?v=<?php echo $cacheVer; ?>"></script>
     <script src="scripts/vendor/bootstrap-notify.min.js?v=<?php echo $cacheVer; ?>"></script>
+<?php if ($auth) { ?>
```

Comment:
Do you want to leave this here or move it to `header_authenticated.php`? It won't be used on the login page.

Reply:
Same as above.